### PR TITLE
Personal/yufei/add perturb doc

### PR DESCRIPTION
**Pull request by moria97**
Add perturnbation document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -357,6 +358,27 @@ To substitute Address data types with a fixed json fragement
 }
 ```
 
+To perturb age fields of Condition resource by sampling noise from a fixed span
```
Comment (ranvijaykumar) [inline README.md:361]: Provide numerical details. 'To perturb age fields of Condition resource by adding random noise having range [-2,2]'

File: README.md [hunk 2]
```diff
@@ -357,6 +358,27 @@ To substitute Address data types with a fixed json fragement
...
+```json
+{
+  "path": "Condition.onset | Condition.abatement as Age",
+  "method": "perturb",
+  "span": 3,
+  "rangeType": "fixed",
+  "roundTo": 0
+}
+```
+To perturb age fields of Condition resource by sampling noise from a proportional span
```
Comment (ranvijaykumar) [inline README.md:371]: Here too, provide numerical details for clarity.

File: README.md [hunk 3]
```diff
@@ -400,6 +422,14 @@ You can specify a fixed, valid value to replace a target FHIR field. For example
 For complex data types, you can provide a fixed json fragment following the [sample rules](#Sample-rules-using-FHIRPath).
 You should provide valid value for the target data type to avoid unexpected errors.
 
+## Perturb method
+With perturbation rule, you can replace specific values with equally specific, but different values. You can choose to sample noise from a fixed range or a proportional range. There are a few parameters that can help you customize the noise amount for different FHIR types.
```
Comment (ranvijaykumar) [inline README.md:426]: Explain with examples what is fixed range and what is a proportional range

File: README.md [hunk 4]
```diff
@@ -400,6 +422,14 @@ You can specify a fixed, valid value to replace a target FHIR field. For example
 For complex data types, you can provide a fixed json fragment following the [sample rules](#Sample-rules-using-FHIRPath).
 You should provide valid value for the target data type to avoid unexpected errors.
 
+## Perturb method
+With perturbation rule, you can replace specific values with equally specific, but different values. You can choose to sample noise from a fixed range or a proportional range. There are a few parameters that can help you customize the noise amount for different FHIR types.
+- [required] **span** A non-negative value representing the random noise range. For *fixed* range type, the noise will be sampled from a uniform distribution over [-*span*, *span*]. For *proportional* range type, the noise will be sampled from a uniform distribution over [-*span* * value, *span* * value].
```
Comment (ranvijaykumar) [inline README.md:427]: Mathematically, span is defined as (max - min) of the range. So, if the user specifies span, the distribution will be [-span/2 , span/2]

File: README.md (hunk 1 above)
Comment (moria97) [reply README.md]: Thanks, updated

File: README.md (hunk 2 above)
Comment (moria97) [reply README.md]: Thanks, updated

File: README.md (hunk 3 above)
Comment (moria97) [reply README.md]: Thanks, updated

File: README.md (hunk 4 above)
Comment (moria97) [reply README.md]: Thanks, updated

File: README.md [hunk 5]
```diff
@@ -400,6 +426,16 @@ You can specify a fixed, valid value to replace a target FHIR field. For example
 For complex data types, you can provide a fixed json fragment following the [sample rules](#Sample-rules-using-FHIRPath).
 You should provide valid value for the target data type to avoid unexpected errors.
 
+## Perturb method
+With perturbation rule, you can replace specific values with equally specific, but different values. You can choose to add random noise from a fixed range or a proportional range. In the [age example](#Sample-rules-using-FHIRPath) above, for a fixed range ```[-3, 3]```, every age is within +/- 2 years of the original value. For a proportional range ```[-0.1*originalAge, 0.1*originalAge]```, every age is within +/- 10% years of the original value.
```
Comment (ranvijaykumar) [inline README.md:430]: Typo... every age is within +/- 3 years.

**Pull request merged**